go/integration_test/interpreter/integration_test.go

```diff
@@ -1109,3 +1109,29 @@ func newMockStateDBForIntegrationTests(ctrl *gomock.Controller) *MockStateDB {
 
 	return mockStateDB
 }
+
+func TestEVMCanSuccessfullyProcessIncorrectCode(t *testing.T) {
+	for _, variant := range Variants {
+		for _, revision := range revisions {
```

Comment:
Is it really required to run this for all revisions, the stats handling is revision independent.

Reply:
This test should not target specifically the bug in the `stats`. It checks a general requirement on all implementations to be able to handle situations in which the PC moves beyond the end of the code.

However, this is a property of the interpreter that is likely revision independent. So the revision loop might indeed be dropped.